Makefile

```diff
@@ -0,0 +1,29 @@
+# Makefile
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
+include common.mk
+SHELL := /bin/bash
+
+RUNTEST=python -m unittest -v -b
+
+ALLMODULES=$(patsubst %.py, %, $(wildcard test_*.py))
+
+all:
```

Comment:
Why not call this something like `tests` instead of `all`?

Reply:
pl. see my response below